storage/redis/peer_store.go

```diff
@@ -743,23 +762,23 @@ func (ps *peerStore) collectGarbage(cutoff time.Time) error {
 					}
 					if mtime <= cutoffUnix {
 						log.Debug("storage: deleting peer", log.Fields{
-							"Peer": decodePeerKey(pk).String(),
+							"Peer": bittorrent.MustParsePeer(peerStr),
```

Comment:
I know it's been like this before, but I dislike that we do the parsing even though debug logging might be disabled...

Reply:
I'd like to switch us to zerolog, which is what I use at work. It's more ergonomic than logrus and would let us drop our own log package because the zerolog API is almost exactly the same as what we've built.